### DISCUSSION TITLE
Creating space between lines

### DISCUSSION THREAD

**Discussion by Artemis-Hound** [JanDeDobbeleer/oh-my-posh#4531] [Troubleshoot] [answered]
Hello, newbie at oh-my-posh - wondering how I'd be able to get some space between blocks in PowerShell in Terminal.
Currently it looks a little crowded on the default theme and I'd prefer if the blocks didn't touch:

![2023-12-07_23-48-26_WindowsTerminal_iK4](https://github.com/JanDeDobbeleer/oh-my-posh/assets/10010491/9bdbeb32-4b7b-4238-8c10-e6fc8cf5c013)

I found the "Line Height" setting in Terminal and was hoping this would fix it, so I tested it with a value of `1.5`:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/10010491/ec24e02e-ff99-4427-b584-134e8fa8d765)

But the end result was very strange and didn't solve my issue:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/10010491/a8e2006e-96f9-4c6c-970f-4b25c74fe16c)

Does anybody know what I might be able to do to solve this?

**Comment by JanDeDobbeleer** [comment, marked as answer]
You can [export the default theme](https://ohmyposh.dev/docs/installation/customize#adjust-a-theme) and use the `newline` property on the first block.
